.drone.yml

```diff
@@ -44,29 +57,54 @@ steps:
 #    - push
 #    - pull_request
 
-- name: Push Image to Quay.io
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind:latest
+- name: maintenance-image-scan
```

Comment:
Can we please rename this to something more meaningful that explains the purpose of the scan.